src/GoogleCloudPubSub/GoogleCloudPubSub.ts

```diff
@@ -238,8 +238,12 @@ export class GoogleCloudPubSub implements GCPubSub {
   private async getOrCreateSubscription(
     name: string,
     topic: Topic,
-    options?: GCSubscriptionOptions,
+    subOptions?: GCSubscriptionOptions,
   ): Promise<Subscription> {
+    const options = {
+      get: { autoCreate: true, ...subOptions?.get },
+      ...subOptions,
```

Comment:
It will always be overwritten if the less precise `subOptions` is after.
```suggestion
      ...subOptions,
      get: { autoCreate: true, ...subOptions?.get },
```